py/samples/google-genai-image/src/image_on_gemini.py

```diff
@@ -14,33 +14,49 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
+"""This sample demonstrates how to use Gemini to describe and draw images."""
+
 import asyncio
 import base64
 import os
 from io import BytesIO
 
 from PIL import Image
 
-from genkit.ai import Genkit, Media, MediaPart, Message, Role, TextPart
+from genkit.ai import Genkit
 from genkit.plugins.google_genai import (
     GoogleGenai,
     google_genai_name,
 )
+from genkit.typing import Media, MediaPart, Message, Role, TextPart
```

Comment:
I'm consolidating these imports under genkit.ai
we seem to be stepping on each other's work
let's coordinate